couchdbkit/schema/base.py

```diff
@@ -130,8 +131,13 @@ def save(self, **params):
         self.validate()
         db = self.get_db()
 
+        for prop_name, prop in self.properties().iteritems():
+            if isinstance(prop, p.DateTimeProperty) and prop.auto_now_add:
+                setattr(self, prop_name, datetime.utcnow())
```

Comment:
curious if you think i should add this logic for for bulk saving. i believe the django implementation would not update last_modified if using bulk_create since it happens in the presave signal (may be a problem for dims/facts that i'll have to address)